### ISSUE TITLE
Host file not updated from vvv-config but from vvv-hosts

### ISSUE THREAD

**Issue by grks** [Varying-Vagrant-Vagrants/VVV#1037]
at the end of provision-site.sh, it still reference vvv-hosts file and use it to add domains to the /etc/hosts on the host. 
````bash
 echo "Adding domains to the virtual machine's /etc/hosts file..."
  find ${VM_DIR} -maxdepth 4 -name 'vvv-hosts' | \
  while read hostfile; do
...
````
However, the default vvv-hosts file was removed in 894bb966f97eecbb9669fddf4da0002e1928f60f

Shouldn't provision-site.sh now rather use vvv-config as source for mapping domains in /etc/hosts? Or should we still keep a vvv-hosts around?

**Comment by LoreleiAurora** [comment]
I like this idea as it gets rid of the need for a double provision.
@jeremyfelt thoughts?

**Comment by LoreleiAurora** [comment]
Fixed in #1053 (And back compatible)
